lib/js/components/Tile/Tile.consts.ts

```diff
@@ -11,15 +11,15 @@ export const TILE_COLORS = {
 	INFO: 'info',
 } as const;
 
-export type TileColors = Value<typeof TILE_COLORS>;
+export type TileColor = Value<typeof TILE_COLORS>;
 
 export const TILE_STATES = {
 	DEFAULT: 'default',
 	DISABLED: 'disabled',
 	LOADING: 'loading',
 } as const;
 
-export type TileStates = Value<typeof TILE_STATES>;
+export type TileState = Value<typeof TILE_STATES>;
```

Comment:
We use those types in WNL but I don't see changes that are made there https://github.com/bethinkpl/wnl-platform/pull/5735
Example of use of one type:
* https://github.com/bethinkpl/wnl-platform/blob/0688106757fb97127c4f22fdde9cdf19f576e66c/resources/assets/js/components/users/aggregatedProfile/ActiveCourses.vue#L26
but as we change a lot of them from the plural form to singular we need to verify if they are not used outside of DesignSystem

Reply:
Good catch - updated wnl-platform: [bethinkpl/wnl-platform@`e0d7d97` (#5735)](https://github.com/bethinkpl/wnl-platform/pull/5735/commits/e0d7d97db90b6df3134acc04554bfa9a62975511)